app/controllers/state_file/questions/w2_controller.rb

```diff
@@ -9,6 +9,7 @@ def edit
 
       def update
         @w2.assign_attributes(form_params)
+        @w2.check_box14_limits = true
```

Comment:
note from review: can we turn on this flag in one place when the import is done?

Reply:
@mmazanec22 I did manage to try out an `after_create` callback on the W2 model to toggle `check_box14_limits` just after creation, which does allow import... but it's a bit of a challenge to get it to *stay* set, I think because the controllers re-fetch them from the database when the forms load. So I think I'd either need to

1- find a way to pass the w2 model instances from one place to another instead of loading from DB
2- add a db column for this and set it
3- keep how i have it, setting in each controller each time

I think the second option is good but is not so much better that I want to add it to this PR.